### PR TITLE
Serve

### PR DESCRIPTION
**Pull request by Mov305**
1. Added the required redux methods
2. Added the API 
3. Added a filter method 
4. Added a Detailed page for every single country
5. Styled the app
6. Fixed the linters
7. Served the App

### CODE REVIEW

File: src/components/Countries/filter.jsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                      className="flex hover:cursor-pointer justify-between rounded-lg px-4 py-2 text-sm text-gray-500 hover:bg-gray-50 hover:text-gray-700 dark:text-gray-400 dark:hover:bg-gray-800 dark:hover:text-gray-300"
+                      role="menuitem"
+                    >
+                      <span>{item.name}</span>
+                      <item.Icon className="inline-block ml-2" />
+                    </li>
+                  ))}
+                </ul>
+              </div>
+
```
Comment (VICTOR-CHUKWU) [inline src/components/Countries/filter.jsx:107]: - [ ] kindly know that this is a mobile view project, in the mobile view of your app the filter drop down is behind the country card. kindly fix this
Screenshot from 2022-10-05 23-19-36

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Mov305 , 
## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/Countries/filter.jsx (hunk 1 above)
Comment (Mov305) [reply src/components/Countries/filter.jsx:107]: Oh! I forgot about that thank you I am gonna fix it right away

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Mov305 ,
# STATUS: APPROVED! 🥇 💯 

done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**